src/patterns/dosdp-patterns/cellCapableOfSensoryPerception.yaml

```diff
@@ -0,0 +1,42 @@
+pattern_name: cellCapableOfSensoryPerception
+pattern_iri: http://purl.obolibrary.org/obo/cl/cellCapableOfSensoryPerception
+
+description: A cell type that is capable of detecting a stimulus that is involved in sensory pereception (eg a receptor cell).
+example: CL_0000006 neuronal receptor cell
```

Comment:
This seems odd

https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000006

`neuron and receptor cell and capable of some detection of stimulus involved in sensory perception`

Do we think this is a good logical definition?

receptor cell is undefined?
https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000197

This seems like it needs review, cc @dosumis